pom.xml

```
@@ -54,27 +54,33 @@
         <placeholderapi.version>2.10.1</placeholderapi.version>
         <githubapi.version>1.0</githubapi.version>
         <!-- Revision variable removes warning about dynamic version -->
-        <revision>${build.version}</revision>
+        <revision>${build.version}-SNAPSHOT</revision>
+        <!-- Do not change unless you want different name for local builds. -->
+        <build.number>-LOCAL</build.number>
         <!-- This allows to change between versions. -->
         <build.version>1.5.0</build.version>
     </properties>
 
     <!-- Profiles will allow to automatically change build version. -->
     <profiles>
         <profile>
-            <!-- Local profile is activated by default. It adds '-SNAPSHOT' at the end of ${build.version} -->
-            <!-- This profile will be used only if develop and master fails. -->
-            <id>local</id>
+            <!-- ci profile is activated if exist environment variable BUILD_NUMBER.  -->
+            <!-- It replaces ${build.number} that is currently '-LOCAL' with correct build number from JENKINS machine.  -->
+            <id>ci</id>
             <activation>
-                <activeByDefault>true</activeByDefault>
+                <property>
+                    <name>env.BUILD_NUMBER</name>
+                </property>
             </activation>
             <properties>
-                <revision>${build.version}-SNAPSHOT</revision>
+                <!-- Override only if necessary -->
+                <build.number>-#${env.BUILD_NUMBER}</build.number>
```

```suggestion
                <build.number>-b${env.BUILD_NUMBER}</build.number>
```
It is a better "convention" to use the letter `b`, which means "build".